### ISSUE TITLE
Issue with Route53 

### ISSUE THREAD

**Issue by eniac111** [clouddrove/terraform-aws-ses#25]
I'm always getting this issue:

```
│ Error: creating Route 53 Record: InvalidChangeBatch: [Tried to create resource record set [name='mydomain.dev.', type='TXT'] but it already exists]
│       status code: 400, request id: xx-xx-xx-xx-xxxx
│ 
│   with module.mydomain_dev_ses.aws_route53_record.ses_verification[0],
│   on ../modules/aws-ses/main.tf line 49, in resource "aws_route53_record" "ses_verification":
│   49: resource "aws_route53_record" "ses_verification" {
│ 
```

The resource doesn't exist. It's probably not iterated right.

Has anyone run into this? I'm still figuring out how to solve it.

Edit: https://github.com/hashicorp/terraform-provider-aws/issues/12322 It's a known issue with the plugin.

**Comment by anmolnagpal** [comment]
@themaniskshah  can you please look into this

**Comment by themaniskshah** [comment]
Hello @eniac111, I appreciate your contribution and taking the time to report the issue in our GitHub repository.  I've fixed the issue with Route53 and released a new version, tagged as `v1.3.2`. You can go ahead and check it out now.
Please let me know if you encounter any further issues, or if you have any additional questions. The issue is now resolved, and I'm closing it. Thank you again for your contribution!
